packages/beacon-node/src/metrics/nodeJsMetrics.ts

```diff
@@ -13,5 +13,7 @@ export function collectNodeJSMetrics(register: Registry, prefix?: string): void
   // - nodejs_gc_runs_total: Counts the number of time GC is invoked
   // - nodejs_gc_pause_seconds_total: Time spent in GC in seconds
   // - nodejs_gc_reclaimed_bytes_total: The number of bytes GC has freed
-  gcStats(register, {prefix})();
+  // `close` must be called to stop the gc collection process from continuing
+  const close = gcStats(register, {collectionInterval: 6000, prefix});
```

Comment:
why is the interval every 6 seconds? Is it just randomly selected or is there some reason for that specific value?